lib/config/options/index.ts

```diff
@@ -2118,12 +2118,12 @@ const options: RenovateOptions[] = [
     subType: 'string',
```

Comment:
The ESLint docs mention a `sort-keys` [^1] configuration option. Can we use that for keys in the `allowedValues` array somehow?

[^1]: [ESLint docs for `sort-keys`](https://eslint.org/docs/latest/rules/sort-keys)

Reply:
Sometimes it's best to have them in a non-alphabetical order

Reply:
I'll leave things as they are now then. 😄